Stylesheets/components/avtar.css

```diff
@@ -0,0 +1,76 @@
+@import url("/stylesheets/main.css");
+
+.center {
+  margin-left: 0%;
+}
+
+.avtar {
+  margin: 0.3rem;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 50%;
+  position: relative;
+  left: 30%;
```

Comment:
when ever you use avtar class, it will always be 30% from the left. 

General suggestion, don't style using float related properties, unless you are using poistion: fixed or position absolute

Reply:
okay . got it